<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QWidget" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>731</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>731</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_email">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>731</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cantFollowed">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cantPosts">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
